app/models/miq_report/generator.rb

```diff
@@ -312,8 +340,8 @@ def generate_basic_results(options = {})
     rbac_opts = options.merge(
       :targets          => targets,
       :filter           => conditions,
-      :include_for_find => get_include_for_find,
-      :references       => get_include,
+      :include_for_find => get_include_for_find_rbac,
+      :references       => get_include_rbac,
```

Comment:
little mixed on whether we need the `get_include` and `get_include_rbac` (same goes for `get_include_for_find{,_rbac}`

Reply:
There are other places in this file that pass in a scope to Rbac:

https://github.com/ManageIQ/manageiq/blob/6f23bdc6b54c638748b7ce925c4971dc7b7595af/app/models/miq_report/generator.rb#L295-L296

In addition to passing in arguments to `:include_for_find` and `:references`.

Not claiming it is "right", just defensively coding here to fix a bug.  "Right" can come later\*.

\*I realize this almost never happens, but since we are blocking others without this fix, I am more inclined to take this stance currently.